includes/scheduler/class-comment.php

```diff
@@ -25,18 +25,7 @@ public static function init() {
 
 		// Comment transitions.
 		\add_action( 'transition_comment_status', array( self::class, 'schedule_comment_activity' ), 20, 3 );
-		\add_action(
-			'edit_comment',
-			function ( $comment_id ) {
-				self::schedule_comment_activity( 'approved', 'approved', $comment_id );
-			}
-		);
```

Comment:
@pfefferle `edit_comment` has a call to `wp_transition_comment_status()` following [a few lines later](https://github.com/WordPress/wordpress-develop/blob/6.7/src/wp-includes/comment.php#L2654-L2654). Do we need a separate callback for that?

Reply:
🤔 I don't think so...

Reply:
🤔 seems to be obsolete then!

sadly I can't remember why I initially added it!

maybe we should check if there is a unit-test that checks for `create` and `update`???

Reply:
Fixed in [cbe3451](https://github.com/Automattic/wordpress-activitypub/pull/1188/commits/cbe345110361da8a489ba840cecfc36996dc4974).